runtime/local/Cargo.toml

```diff
@@ -57,6 +57,8 @@ frame-system-rpc-runtime-api = { git = "https://github.com/paritytech/substrate"
 pallet-transaction-payment-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.27", default-features = false }
 
 # Astar pallets
+chain-extension-trait = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.27", default-features = false }
+dapps-staking-chain-extension = { git = "https://github.com/AstarNetwork/astar-frame", branch = "polkadot-v0.9.27", default-features = false }
```

Comment:
I think we should move chain extensions under its own category instead of the pallet category

Reply:
This repo should be fine, we agreed initially to use it for such purposes.

This comes especially into play when precompiles/chain-extensions need to be changed as the pallet changes. So it makes sense to keep them next to each other.